pq-crypto/s2n_pq.c

```diff
@@ -82,19 +93,62 @@ bool s2n_cpu_supports_sikep434r2_asm() {
 #endif /* defined(S2N_SIKEP434R2_ASM) */
 }
 
+void s2n_bike_r3_x86_64_opt_init() {
```

Comment:
Can we maintain the function signature convention that was started with sike?

```
S2N_RESULT s2n_try_enable_bikel1r3_optimizations(void);
```

Reply:
this function is not trying to enable anything, it's just initializing the flags. But I added new functions that try to enable specific optimizations:
`S2N_RESULT s2n_try_enable_bike_r3_opt_pclmul(void);
S2N_RESULT s2n_try_enable_bike_r3_opt_avx2(void);
S2N_RESULT s2n_try_enable_bike_r3_opt_avx512(void);
S2N_RESULT s2n_try_enable_bike_r3_opt_vpclmul(void);`

They are used in the new unit tests that I also added with the latest commit.